styles/globals.css

```diff
@@ -15,24 +15,36 @@
 
   body,
   #__next {
-    @apply flex min-h-full flex-col text-base leading-relaxed;
+    @apply flex min-h-full flex-col text-base text-uog-color-body-copy leading-relaxed;
     font-size: 1.8rem;
   }
 
   #__next {
     @apply flex-1;
   }
 
-  h1,
-  h2,
-  h3,
-  h4,
-  h5,
-  h6 {
+  h1, h3, h4, h5, h6 {
+    @apply text-uog-color-black;
+  }
+
+  h2 {
+    @apply text-uog-color-red;
+  }
+
+  h1, h2, h3, h4, h5, h6 {
     line-height: 1.4;
```

Comment:
To Do item for the gus side of things:I think I'll need to create a global stylesheet update to change the line-height for the headings to 1.4 (I believe the nextJS branch has them correct, but the gus / global styles have it at 1.2)

Reply:
Resolving as this is a to do item in another repo